lib/index.js

```diff
@@ -213,7 +213,9 @@ Segment.prototype.normalize = function(msg) {
       ctx.traits.crossDomainId = crossDomainId;
     }
   }
-  if (query) ctx.campaign = utm(query);
+  if (query && !ctx.campaign) {
```

Comment:
We don't want to override campaign if a user is providing one. This is fairly standard in most of our client libraries, including this one, no?

1. https://github.com/SpainTrain/analytics.js-integration-segmentio/blob/bff98739474c1ce92809601cc4f76bb4acc6fad5/lib/index.js#L210
2. https://github.com/SpainTrain/analytics.js-integration-segmentio/blob/bff98739474c1ce92809601cc4f76bb4acc6fad5/lib/index.js#L207